### ISSUE TITLE
Add JS code needed for SEPA method in the blocks checkout

### ISSUE THREAD

**Issue by superdav42** [Automattic/woocommerce-payments#1470]
Probably this should be done after #1469 to glean from any lessons learned from the classic jquery implementation.

The sepa.js file is https://github.com/Automattic/woocommerce-payments/pull/1452 should give an idea of what will be needed by blocks checkout to get SEPA working. An implementation should be chosen that avoids code duplication so that new payment methods can be easily added.

**Comment by bborman22** [comment]
The move to UPE closes this issue as no longer needed.
